src/utils/ddb/ddb_commands.c

```diff
@@ -1013,7 +1013,7 @@ ddb_run_feature(struct ddb_ctx *ctx, struct feature_options *opt)
 		ctx->dc_write_mode = false;
 
 	if (!ctx->dc_write_mode && !opt->show_features)
-		return -DER_INVAL;
+		return -DER_NO_PERM;
```

Comment:
Seems some CI test failures because of the new test cases?

Reply:
Yup.